src/client/components/utils/ToggleButton.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+class Toggle extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {isToggleOn: true };
+        this.handleClick = this.handleClick.bind(this);
+    }
+
+    handleClick() {
+        this.setState(function(prevState) {
+            return {isToggleOn: !prevState.isToggleOn};
```

Comment:
@vitkuz why do you need previous state?
Doesn't code below work for you?
```
this.setState({
  isToggleOn: !this.state.isToggleOn
})
```

Reply:
done